snuba/subscriptions/worker.py

```diff
@@ -189,7 +191,9 @@ def callback_func(
                                 "consistent": str(is_consistent_query),
                                 "primary": primary_result,
                                 "other": result.result,
-                                "query": request.query,
+                                "query": request.query.get_all_expressions(),
```

Comment:
Why not using the formatter we built specifically for these cases 
https://github.com/getsentry/snuba/pull/1600/files#diff-5d8fe413b73c6027fa896e84fd6f6952ae8b897418c903944ecfca36f65cc625R118

Reply:
The formatter works on a physical Clickhouse query type, here I have the raw query from the request so this is easier